conf/activemq.xml

```diff
@@ -0,0 +1,176 @@
+    <!--
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.